### ISSUE TITLE
Start problems

### ISSUE THREAD

**Issue by GerhardKreuzer** [Apoc182/local_daikin#28]
Hi everyone,
I also want to integrate FTXAxxAW boxes, but no glue how to start.

Situation:
HA well running and a WiFi well working (for PC, Laptops and ESPHome devices, cloudefree Shellys ...). Laptop and other PCs with working internet connection behind a fire wall. 
10 Daikin boxes with WiFi modules.

Target:
10 Daikin boxes controlled by HA, cloudfree.
First step (as I thought) is the setup of the Daikin boxes to be part of my WLan, static IP addresses preferred.

Outcome after half day playing around and now totaly frustrated:
I switched a box with the remote control to start a WLan. I also have the SSID and Key on a sticker. I see the net with the correct SSID and I connected my smartphone to this net using the specified key. Smartphone told me, 'connected, without internet' which is ok, cause it is a very tiny net with only two devices connected, the Daikin box and my smartphone (LTE disabled).
I used two different apps from Daikin, one described in the printed manual, the other (newer?) following a link displayed in the first app, but no chance.

I do it with different boxes and more than once, no connection.

As I read here, that you control the box with HA, please give me some hint. How can I configure this dammed boxes to be part of my net and not part of a Daikin cloud (which also doesn't work out. The newer app switches networks inbetween, but also doesn't worked out)

Thanks a lot.

With best regards

Gerhard

**Comment by Apoc182** [comment]
HI @GerhardKreuzer, you would need to get the units working with their native app on your phone on your network before attempted this HA integration. Unfortunately, this isnt something I can help you with. I read the manufacturers instructions to work this out, maybe you can find what you need there?

**Comment by GerhardKreuzer** [comment]
Hi Apoc182,
thanks a lot (I am an idiot) and entered the wrong WiFi key as I tried to add the Daikin box to our network , too many different locations.
Now its solved and one box is in the Daikin cloud. This isn't local, but one step after the other.

I installed the 'local-daikin' integration and I know, I have to enter the IP-address of the box(s) in the configuration.yaml file I guess, but there were no more entities to control the box. What do I miss now?

And, after blocking the Daikin box in my firewall  (no cloud connection anymore) will be the system continuing working or not? Unfortunately your integration has a lable 'Needs Internet'. ((Or does this mean needs local network?))

Thanks a lot for helping.

With best regards

Gerhard

**Comment by GerhardKreuzer** [comment]
Hi Apoc182,
any news?

**Comment by Apoc182** [comment]
Sorry @GerhardKreuzer . My integration does not require internet and to my knowledge that Daikin app does not require internet unless you register it to be used outside of your local network. Have you attempted to do this yet?

**Comment by GerhardKreuzer** [comment]
Hi Apoc182,
I used the Daikin App to bring the unit into my network. Now I have an IP address and I am able to ping it. 
For this step you have to use the Onecta app and the cloud, I think. Or is there another way?
I also entered the lines in the configuration.yaml, using this IP.
Whats the next step? Is there a frontend and how to enter a name for the unit and the entities?
With best regards
Gerhard

**Comment by GerhardKreuzer** [comment]
Hi Apoc182,
I configured one of the splits as you described using the configuration.yaml, ok.
I read the documentation of the 'Climate' building block by HA but I don't have any idea how to use this two things ...

Can you provide a very simple example, maybe switch the box on and off? 
Should I use 'Climate' or not? If I should use it, how to link them together?

If I have more than one box, how can I set names? How can I refer to this names?

Sorry, but I don't understand the idea behinde the 'Climate' building block and the devices (split boxes).

With best regards

Gerhard

**Comment by GerhardKreuzer** [comment]
Hi,
to demonstrate my problems, here what I do:
I created a button to toggle the on off state of one of my splits:
show_name: true
show_icon: true
type: button
tap_action:
  action: toggle
entity: script.ac_on_off_test

Here is the script:

action: climate.toggle
metadata: {}
data: {}

Yes, I know, that I have to point to a specific device, but from the other end, the local-daikin integration, I didn't get anything, cause this looks like:

climate:
  - platform: local_daikin
    ip_address: 192.168.x.y

And yes, I can successful ping this address, so this part should ok.

Now my question:
How can I close this 'logical gap', or where can I find information, how to set up such an scenario?

And yes, I have more than one split to control independently!

Thanks a lot for helping.

With best regards

Gerhard
PS: If I can control the splits by scrips/automations, I can setup a control loop, for sure.

**Comment by GerhardKreuzer** [comment]
Any news here?
I still can't find anything of my splits integration in my HA instance.

**Comment by Apoc182** [comment]
Hi Gerhard. This is outside of my knowledge. The integration should just work when you assign IPs in the config. If you are looking for help with setting up scripts/automations to leverage the available entities, this question might be better asked to the HA community in general.

Thanks.

**Comment by GerhardKreuzer** [comment]
Yes, I do so, one IP, but I cant find any entities or other signes in HA of this integration, that my problem .... Is there an other component/integration ... to install?
I found the .py scripts, but no idea, were the used/executed
Sorry

With best regards
Gerhard

**Comment by Apoc182** [comment]
![image](https://github.com/user-attachments/assets/55445bfd-70b4-4141-91f5-e50ea85b721a)

![image](https://github.com/user-attachments/assets/7da90f1e-c2b5-4750-8c57-3c0dc6db249a)

I'm not sure what else you need assistance with. Are you new to Home Assistant?

**Comment by GerhardKreuzer** [comment]
Hi,
thanks for your help.
I am pretty new and constantly learning.
I get exactly what you show on the screen shots. 
But how can I access the functions of my splits?
I just want to switch on/off as a first step.
You also show, how to add more than one IP-Address == split, so how can I distinguish between this boxes later?

I think I miss something and installed your package (which probably extends another) only.

With best regards
Gerhard

**Comment by GerhardKreuzer** [comment]
Good morning, any news?

**Comment by Apoc182** [comment]
Hi Gerhard. There is no more information in the integration. You can simply go to a dashboard and add your entity `climate.local_daikin` and you will have all the controls on the dash.

If you want to toggle one on and off, here is a very simple automation i threw together to illustrate:

```
description: ""
mode: single
triggers:
  - trigger: time
    at: "00:00:00"
conditions: []
actions:
  - action: climate.toggle
    metadata: {}
    data: {}
    target:
      entity_id: climate.local_daikin
```
![image](https://github.com/user-attachments/assets/29ad74e2-5815-4bc4-93e3-9502468bf86f)


As for running two, I only have one unit so am not sure how they are differentiated. I assume it would automatically appends an `_1` ,  `_2` etc. to the end and you can just rename the device.

![firefox_9ftcjKwNyW](https://github.com/user-attachments/assets/f10e96f7-87df-40b3-827e-641a799efe52)

**Comment by GerhardKreuzer** [comment]
Hi,
sorry, I didn't understand how to do.
I go to a dashboard. I added a thermostat tile. I dropped down the entity list, no daikin_local. I typed in 'climate.local_daikin' no hit.
Maybe something went wrong.

Should I do a re-install?
What is the target?
How can I check that the installation works?

With best regards

Gerhard

**Comment by Apoc182** [comment]
Perhaps. You will need this in your config yaml

```
climate:
  - platform: local_daikin
    ip_address: 
      - X.X.X.X
```

If the integration is running correctly, you will see in the gif i shared that there is 1 entity under the local daikin package. If there is not, check your logs. Something must be incorrect with the IP address or hardware.

**Comment by GerhardKreuzer** [comment]
I have it:
![grafik](https://github.com/user-attachments/assets/984c0eda-e36a-4368-9d7d-e89fc46f7b73)

But no traces of 'local_daikin in any of the HA parts.
I can ping the split using this address, so network ist ok.
See my posting above, I added screenshots.
With best regards
Gerhard

**Comment by Apoc182** [comment]
Do you have issues in your log? Your unit may not be compatible or the firmware might be different.

**Comment by GerhardKreuzer** [comment]
I never saw a 'local_daikin' entry in the 'LogViewer' of my HA. Is there a chance to see the scrips running or some other chance to see something?
